docker-compose/monitor/docker-compose-v2.yml

```diff
@@ -0,0 +1,54 @@
+version: "3.5"
+services:
+  jaeger:
+    networks:
+      - backend
+    # TODO: Need to use the local image here while making changes 
+    image: jaegertracing/jaeger
+    volumes:
+      - "./jaeger-ui.json:/etc/jaeger/jaeger-ui.json" # Do we need this for v2 ? Seems to be running without this.
```

Comment:
it has this:
```
  "monitor": {
    "menuEnabled": true
  },
```
I don't remember if it's the default. We already have a Capabilities exchange protocol between backend and UI, ideally it would just use that and not require explicit config.